cmp/grid/GridModel.js

```diff
@@ -649,6 +650,10 @@ export class GridModel {
         return c.children ? new ColumnGroup(c, this) : new Column(defaultsDeep({}, c, this.colDefaults), this);
     }
 
+    autoSizeAllColumns() {
```

Comment:
I think we should consider changing this to `autoSizeColumns` which can accept a single or list of `Column` objects or column ids, and it can default to `this.getLeafColumns()`.

Pretty sure we want to use `getLeafColumns()` here instead of the `columns` list as that contains column groups, so we are probably skipping autosizing of their child columns - unless ag-grid is handling that for us when we ask to autosize a column group.